File: projeto-integrador/vue-inforental/src/components/Chamado.ts
export class Chamado {
  constructor(
    public id: number,
    public Cnpj: string,
    public DataAbertura: Date,
    public DataFechamento: Date | null,
    public Equip: string,
    public DescProblema: string,
    public DescSolucao: string | null,
    public Status: boolean
  ) {}
}

File: projeto-integrador/InfoRental/Models/Chamado.cs
namespace InfoRental.Models;

public class Chamado
{
    public int Id { get; set; }
    public DateTime DataAbertura { get; set; } = DateTime.Now;
    public DateTime? DataFechamento { get; set; }
    public string NSerie { get; set; } = null!;

    //relacionamento 1-1 com Equipamento
    public Equipamento Equip { get; set; } = null!;
    public string DescProblema { get; set; } = null!;
    public string? DescSolucao { get; set; }

    // chamado false = aberto pois o problema não foi resolvido
    // chamado true = fechado pois o problema foi resolvido
    public bool Status { get; set; } = false;
}
